meson.build

```diff
@@ -330,8 +348,8 @@ tar = find_program('tar', required : true)
 bzip2 = find_program('bzip2', required : true)
 gzip = find_program('gzip', required : true)
 xz = find_program('xz', required : true)
-dot = find_program('dot', required : true)
-lsof = find_program('lsof', required : true)
+dot = find_program('dot', required : false)
+lsof = find_program('lsof', required : false)
```

Comment:
lsof is required by libnixstore iirc..?

Reply:
So perhaps didn't do this quite right, but it just needs it an runtime, and will do a PATH lookup then if it doesn't find it.